README.md

```diff
@@ -3,7 +3,17 @@
 This repo contains the Docker file used in the [AiiDAlab](https://www.materialscloud.org/aiidalab).
 
 Docker images are available from Dockerhub via `docker pull aiidalab/aiidalab-docker-stack:latest`.
-See [aiidalab/aiidalab-docker-stack](https://hub.docker.com/repository/docker/aiidalab/aiidalab-docker-stack) for a list of available tags.
+
+The available tags and their meaning:
+
+- `master` tag corresponds to the image build on top of the `master` branch.
+- `develop` tag  corresponds to the image build on top of the `develop` branch.
+- Version number (e.g. `21.12.0`) corresponds to the tagged commit on GitHub.
+- `latest` tag corresponds to the latest released tag.
```

Comment:
I suggest to:
1. Reorder this by order of "importance". I would say that the `latest` tag is probably most relevant to the majority of users.
2. Combing the branch tags, since they have the same meaning.

I have also simplified and streamlined the description language a little bit.

```suggestion
- `latest` –  the latest tagged release
- `<version>` – a specific tagged release, example: `21.12.0`
- `master`/`develop` – the latest commit on the corresponding branches with the same name
```